### PR TITLE
UG-885 Implement config, styles and behaviour for Academia sharing message

### PR DESCRIPTION
**Pull request by jamesnicholls**
Adds config, styles, and client-side behaviour for the Academia sharing message. The message is lazy, and retrieves it's HTML structure from next-messaging-guru (see https://github.com/Financial-Times/next-messaging-guru/pull/178).

In order to test:
1. Set the `messageSlotBottom` flag to the variant `AcademiaCTA`
2. Set the `academicCtaTest` flag to `variant`
3. Checkout the associated [next-messaging-guru PR](https://github.com/Financial-Times/next-messaging-guru/pull/178) and run using `make run`
4. Checkout this PR and run the app using `make demo-with-guru`
5. Go to https://local.ft.com:5005/
6. The message should appear as per [the designs](https://www.figma.com/file/09mlE9M0XVG5fIZy99tls2/(Final)-Professor-Engagement)

### CODE REVIEW

**Review by serena97 — DISMISSED** [review]

**Review by leannecornish-ft — COMMENTED** [review]
Looks good 👍  I just have one question - on mobile view, the copy link button is now quite large as the url has changed, just checking we are ok with that from a style perspective?
Screenshot 2022-06-10 at 18 53 22

**Comment by jamesnicholls** [comment]
> Looks good 👍 I just have one question - on mobile view, the copy link button is now quite large as the url has changed, just checking we are ok with that from a style perspective?

Yeah, good question. I had a concern about that before I left on holiday but forgot to check with Tasha and/or Dilyana. Will check with them now. Thanks for the review.

**Review by leannecornish-ft — APPROVED** [review]
thanks James, looks good to me 👍

File: client/components/bottom/academia-cta/main.scss [hunk 1]
```diff
@@ -77,6 +77,10 @@
 			background-origin: border-box;
 			text-decoration: none;
 			padding-right: 20px;
+			-webkit-box-orient: vertical;
```
Comment (serena97) [inline client/components/bottom/academia-cta/main.scss:80]: question: are we sure we want to use `-webkit-box-orient` here? the docs https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient mentions that it has been deprecated and to use flexbox instead

Comment (jamesnicholls) [reply client/components/bottom/academia-cta/main.scss:80]: Oh good spot. Unfortunately, `-webkit-line-clamp` doesn't work with `display:flex`, so I'll have to find a different approach.

**Review by serena97 — APPROVED** [review]

**Pull request merged**